### PR TITLE
Tenants routes

### PR DESCRIPTION
**Pull request by DaveTrost**
Update table properties and create routes for the Tenants resource

- Properties (update TenantModel):
  - [x] firstName + lastName
  - [x] phone
  - [x] staffIDs - a lookup table (supports a many-to-many relationship between tenants and the social workers
  - [x] propertyID - a foreign key
  - [x] tenantID - this is an old property: remove it
  - [x] landlordID - this is an old property: remove it
on this)
- Routes (create Tenants-Resource):
  - [x] GET: get a tenant by id
  - [x] POST: add a tenant
  - [x] PUT: update the tenant information
- [x] update the seed_db function to include a initial tenant
- [x] update the README with the new routes and other info

### CODE REVIEW

File: resources/tenants.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    parser = reqparse.RequestParser()
+    parser.add_argument('firstName',type=str,required=True,help="This field cannot be blank.")
+    parser.add_argument('lastName',type=str,required=True,help="This field cannot be blank.")
+    parser.add_argument('phone',type=str,required=True,help="This field cannot be blank.")
+    parser.add_argument('propertyID',required=False,help="This field can be provided at a later time.")
+    parser.add_argument('staffIDs',action='append',required=False,help="This field can be provided at a later time.")
+    
+
+    def get(self, tenant_id=None):
+        # The get all endpoint is useful for development. Disable before production??
```
Comment (Chris-Boe) [inline resources/tenants.py]: We'll still need the get all endpoint for the Manage Tenants page 👍

**Review by Chris-Boe — DISMISSED** [review]
Could delete the comment about the Get all endpoint but otherwise looks great! Tested using your front end PR branch https://github.com/codeforpdx/dwellingly-app/pull/176

**Comment by Chris-Boe** [comment]
Also if you want, you can always rebase to combine cleanup commits and fix issues in previous commits - rewriting history 😉

**Comment by DaveTrost** [comment]
> Also if you want, you can always rebase to combine cleanup commits and fix issues in previous commits - rewriting history 😉

This will be useful in the future. I played around with rebasing and then realized that I could not change commits that had already been published, so ... a good learning expedition, and, sorry, but you'll just have to deal with my micro-commits this time!

**Review by htharker42 — APPROVED** [review]
Looks great, Dave! Tested via postman

**Pull request merged**